source/administration/bulk-export-data.rst

```diff
@@ -14,15 +15,17 @@ You can export the following data types:
 - Posts' Replies
 - Posts' Reactions
 - Custom Emoji
-- Direct Message Channels 
+- Direct Message Channels
 - Direct Message Posts
 
-Configuration for exporting specific areas of the server, exporting additional types of posts, permissions schemes, file attachments, webhooks and bot messages are not yet supported. Deleted objects are also not yet supported.  
-
-For requests to add additional attributes or objects to our exporter, please add a feature request on our `feature idea forum <https://mattermost.uservoice.com/forums/306457-general>`__.  
+.. note:: 
+  Configuration for exporting specific areas of the server, exporting additional types of posts, permissions schemes, file attachments, webhooks, and bot messages are not yet supported. Deleted objects are also not yet supported.
```

Comment:
```suggestion
  Configuration for data types such as exporting specific areas of the server, exporting additional types of posts, permissions schemes, file attachments, webhooks, and bot messages is not yet supported. Deleted objects are also not yet supported.
```

Reply:
@svelle, did I get the tense right in my suggestion?

Reply:
I think so, this part wasn't originally written by me. But from my understanding this is what it's supposed to say.